datasets/ua/war_sanctions/crawler.py

```diff
@@ -140,12 +145,24 @@ def crawl_vessel(context: Context, details_container, link):
     vessel.add("description", f"Cases of AIS shutdown: {ais_shutdown}")
     vessel.add("description", f"Calling at russian ports: {ru_ports}")
     vessel.add("description", f"Visited ports: {ports}")
+    vessel.add("description", f"Builder (country): {build_country}")
+    vessel.add("description", f"Flags (former): {flags_former}")
     vessel.add("callSign", data.pop("Call sign"))
     vessel.add("flag", data.pop("Flag (Current)"))
     vessel.add("mmsi", data.pop("MMSI"))
-    vessel.add("topics", "sanction")
+    vessel.add("buildDate", data.pop("Build year"))
     for web_resource in data.pop("Web resources"):
         vessel.add("sourceUrl", web_resource)
+    for name in h.multi_split(data.pop("Former ship names"), [" / "]):
+        vessel.add("previousName", name)
+    vessel.add(
+        "description", f"Commercial ship manager (IMO / Country / Date): {com_manager}"
+    )
+    vessel.add(
+        "description",
+        f"Ship Safety Management Manager (IMO / Country / Date): {safety_manager}",
+    )
+    vessel.add("topics", "sanction")
```

Comment:
is this data that wants to be a relationship? Otherwise let's skip it and avoid semi-structured messes in `description` :)